docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -0,0 +1,470 @@
+# Component Port
+
+Tizen components of the component-based applications can communicate with each other using component ports. Components can send and receive serializable objects through component port communication.
+
+The main feature of the `Tizen.Applications.ComponentBased.ComponentPort` class include:
+- Managing a component port
+  You can set up the component ports to [send and receive requests](#port) between components of component-based application with the `Tizen.Applications.ComponentBased.ComponentPort` class.
+
+- Use component task
```

Comment:
Just for consistency across the pages suggesting this:
```suggestion
- Using component task

Reply:
Done